charts/node-analyzer/values.yaml

```diff
@@ -0,0 +1,369 @@
+# Default values for Sysdig Monitor and Secure Helm package.
+image:
+  # This is a hack to support RELATED_IMAGE_<identifier> feature in Helm based
+  # Operators
+  #
+  # As long as I don't want to people to use this, I will keep it undocumented
+  overrideValue:
+
+  registry: quay.io
+  repository: sysdig/agent
```

Comment:
`repository` and `tag` here can be either removed or maybe changed to the nodeAnalyzer values